support/ci/deploy.sh

```diff
@@ -27,16 +27,28 @@ wget -O hab.tar.gz "${HAB_DOWNLOAD_URL}"
 tar xvzf ./hab.tar.gz --strip 1 -C ${BOOTSTRAP_DIR}
 
 # kick off the mac unstable build
-echo "Kicking off the unstable mac build"
+echo "Kicking off the $BINTRAY_REPO mac build"
 var_file=/tmp/our-awesome-vars
 mac_builder=admin@74.80.245.236
 
+#!/bin/bash
```

Comment:
We probably don't need this line again, do we?  It's already at the top of this script.